packages/aws-cdk-lib/aws-codebuild/lib/fleet.ts

```diff
@@ -0,0 +1,164 @@
+import { Construct } from 'constructs';
+import { CfnFleet } from './codebuild.generated';
+import { ComputeType } from './compute-type';
+import { EnvironmentType } from './environment-type';
+import { Arn, ArnFormat, IResource, Resource, Token } from '../../core';
+
+/**
+ * Construction properties of a CodeBuild {@link Fleet}.
+ */
+export interface FleetProps {
+  /**
+   * The name of the Fleet.
+   *
+   * @default - CloudFormation generated name
+   */
+  readonly fleetName?: string;
+
+  /**
+   * The number of machines allocated to the compute ﬂeet.
+   * Deﬁnes the number of builds that can run in parallel.
+   *
+   * Minimum value of 1.
```

Comment:
Is there a max value?

Reply:
It's an [adjustable quota](https://docs.aws.amazon.com/codebuild/latest/userguide/limits.html#fleet-limits), so yes there's a max value, but I can't list it here 😄